### PR TITLE
Feature/login btns

### PR DESCRIPTION
**Pull request by clari182**
- Adds login and signup in the header and sidebar sections
https://deploy-preview-1108--staging-aiid.netlify.app/

### CODE REVIEW

File: site/gatsby-site/src/components/loginSignup/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { useUserContext } from 'contexts/userContext';
+import { Button, Tooltip } from 'flowbite-react';
+import React from 'react';
+import { Trans } from 'react-i18next';
+
+const LoginSinup = ({ className = '', logoutClassName = '', loginClassName = '' }) => {
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/loginSignup/index.js:6]: typo

File: site/gatsby-site/src/components/sidebar/index.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import { ExternalLink } from 'react-feather';
 import config from '../../../config';
 import QuickAccess from 'components/discover/QuickAccess';
 import { Trans } from 'react-i18next';
+import LoginSinup from 'components/loginSignup';
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/sidebar/index.js:8]: :D

File: site/gatsby-site/src/components/sidebar/index.js [hunk 3]
```diff
@@ -124,6 +125,12 @@ const SidebarLayout = ({ collapse, setNavCollapsed }) => {
           }
         })}
       </ul>
+      <div className="flex items-center justify-center w-full py-3 border-t">
+        <LoginSinup
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/sidebar/index.js:129]: :D

File: site/gatsby-site/src/components/ui/Header.js [hunk 4]
```diff
@@ -9,6 +9,7 @@ import config from '../../../config.js';
 
 import Sidebar from '../sidebar';
 import LanguageSwitcher from 'components/i18n/LanguageSwitcher';
+import LoginSinup from 'components/loginSignup';
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/ui/Header.js:12]: :D

File: site/gatsby-site/src/components/ui/Header.js [hunk 5]
```diff
@@ -174,6 +175,11 @@ const Header = () => {
                     onClick={() => setNavCollapsed(!navCollapsed)}
                   />
                 </div>
+                <LoginSinup
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/ui/Header.js:178]: :D

**Review by cesarvarela — CHANGES_REQUESTED** [review]
Looks good. mergeable after fixing the typo :D

**Comment by clari182** [comment]
@cesarvarela typo fixed.
I retriggered the deploy and it's all green now: https://app.netlify.com/sites/staging-aiid/deploys/632a1d5df7d1e70f83981fc3
Looks like Github doesn't update the latest deploy preview if it's retriggered from Netlify

**Comment by clari182** [comment]
Closing this. Duplicate of #1163

**Pull request closed without merging**